src/test/resources/example/example_four_groups.yml

```diff
@@ -12,12 +12,12 @@ groups:
 items:
 
   - identifier: user
-    name: User
```

Comment:
This is a rename-refactoring artifact. Unfortunately Intellij tries to be clever. Please revert